ngraph/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - jlewi
```

Comment:
No need to list me and Michelle as approvers. That will just assign us extra reviews.